src/main/java/com/eternalcode/parcellockers/configuration/ConfigurationManager.java

```diff
@@ -0,0 +1,41 @@
+package com.eternalcode.parcellockers.configuration;
+
+import net.dzikoysk.cdn.Cdn;
+import net.dzikoysk.cdn.CdnFactory;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ConfigurationManager {
+
+    private final Cdn cdn = CdnFactory
```

Comment:
to możesz zrobić private static final [...], oraz nazwa fielda z dużych liter bo to static.